bcgov/foi-flow
# Flask upgrade + pytest fixes.

Flask upgrade + pytest fixes

Sanity Check

- Create a request
- Sync axis
- Watch/UnWatch Request
- Upload files
- Add comment
- State transition (Open->CFR)
